woocommerce-services.php

```diff
@@ -380,7 +371,8 @@ public function pre_wc_init() {
 				return;
 			}
 
-			add_action( 'rest_api_init', array( $this, 'tos_rest_init' ) );
+			add_action( 'rest_api_init', array( $this, 'rest_api_init' ) );
+			add_action( 'woocommerce_init', array( $this, 'schedule_service_schemas_fetch' ) );
```

Comment:
Just so I don't forget amidst discussions about TOS - we can't fetch schemas without some sort of consent first. I know we're still working that out, but as things currently sit I don't think this is OK.